sormas-app/app/src/main/java/de/symeda/sormas/app/person/SelectOrCreatePersonDialog.java

```diff
@@ -72,7 +72,7 @@ public static void selectOrCreatePerson(final Person person, final Consumer<Pers
 		}
 
 		personDialog.setPositiveCallback(() -> {
-			if (personDialog.getSelectedPerson() != null && !personDialog.getSelectedPerson().getUuid().equals(person.getUuid())) {
+			if (personDialog.getSelectedPerson() != null) {
```

Comment:
`personDialog.getSelectedPerson().getUuid().equals(person.getUuid())` should never be true. If the person already existed before, person selection should be skipped.
If - under any circumstances - a person with the same UUID is selected from a list of already existing ones, a merge should take place here.